### ISSUE TITLE
python package makes reference to missing ParticleStats_TrackCompare.py

### ISSUE THREAD

**Issue by carandraug** [darogan/ParticleStats#6]
The python packages requires a file `scripts/ParticleStats_TrackCompare.py`:

```
    scripts=[ "scripts/ParticleStats_TrackCompare.py", "scripts/ParticleStats_Compare.py", "scripts/ParticleStats_Directionality.py", "scripts/ParticleStats_Kymographs.py", "scripts/ParticleStats_ROI.py" ],
```

Which is not present on the package:

```
$ md5sum ParticleStats-0.3.tar.gz 
5b22ebec256a812e237d5122127cbde8  ParticleStats-0.3.tar.gz
$ tar -tvzf ParticleStats-0.3.tar.gz | grep TrackCompare
$
```

 or the whole history of the repository:

```
$ git log --all --pretty=format: --name-only --diff-filter=A | grep TrackCompare
[again no output]
```

**Comment by darogan** [comment]
TrackCompare became TrackAlign which is now in a separate repo.

Are you interested in comparing the output of Tracking algorithms?

**Comment by carandraug** [comment]
> Are you interested in comparing the output of Tracking algorithms?

Not yet. For now, I'm just interested in installing the python package. This issue, together with issue #5, makes the package installation fail.

What I did was to add the c file from the repo to the root of the package and remove the reference to TrackCompare from `setup.py`. That allows to install it but I have no way to actual check if that's correct.

I wrote a `Makefile` that prepares a release of the python package from the development sources. However, it would have been a lot simpler if I can just change the structure of the files in the repo.
